MatthewKelly12/here_we_grow_django_py
# Remove python environment

## Link to Ticket
Closes
 
## Description of Proposed Changes
-added myvenv to .gitignore
-removed virtual environment from remote repo

## Steps to Test

Outline the steps to test

```
git fetch --all
git checkout remove_python_environment
open project in editor

```

1. verify file structure in editor

## Impacted Areas in Application

- development environment
